conftest.py

```diff
@@ -0,0 +1,3 @@
+def pytest_configure(config):
+    from twisted.internet import selectreactor
+    selectreactor.install()
```

Comment:
The integration tests passed (this time) with this addendum, but locally when I ran the integration tests in a Debian container, they would mostly, but not always fail somewhere in the middle of `robot.py`.  I don't know if this chunk of code exposes, shifts the blame to, or introduces this.